### DISCUSSION TITLE
Referencing existing files when creating records

### DISCUSSION THREAD

**Discussion by xaqbr** [pocketbase/pocketbase#3019] [Q&A]
Hello, is there a way to reference existing files when creating records? For example, I created Record A with a file on one field, and now I want to create Record B with the same file used in Record A. Is there a way I can link to the same file without having to re-upload?

**Comment by manishonc** [comment]
> Hello, is there a way to reference existing files when creating records? For example, I created Record A with a file on one field, and now I want to create Record B with the same file used in Record A. Is there a way I can link to the same file without having to re-upload?

You can create a separate collection for files
Then upload your files there then you can use relation to link that file to any new records.
